test/simulator/abilities/flowergift.js

```diff
@@ -0,0 +1,58 @@
+var assert = require('assert');
+var battle;
+
+describe('Flower Gift', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+	it('should boost allies\' Attack and Special Defense stats', function () {
+		battle = BattleEngine.Battle.construct('battle-flowergift-boost', 'doublescustomgame');
+		battle.join('p1', 'Guest 1', 1, [
+			{species: "Cherrim", ability: 'flowergift', moves: ['healbell']},
+			{species: "Snorlax", ability: 'immunity', moves: ['healbell']}
+		]);
+		battle.join('p2', 'Guest 2', 1, [
+			{species: "Blissey", ability: 'serenegrace', moves: ['healbell']},
+			{species: "Blissey", ability: 'serenegrace', moves: ['healbell']}
+		]);
+		battle.commitDecisions();
+		var cherAtk = battle.p1.active[0].getStat('atk');
+		var cherSpd = battle.p1.active[0].getStat('spd');
+		var baseAtk = battle.p1.active[1].getStat('atk');
+		var baseSpd = battle.p1.active[1].getStat('spd');
+
+
```

Comment:
No! The extra line doesn't go here! It goes in between each of your `it` test cases! Now here has one too many!

Look at the other test files to get a sense of what I'm talking about.